src/tooltip/tooltip.tsx

```diff
@@ -70,8 +73,9 @@ const Tooltip: React.FunctionComponent<Props> = (props) => {
     ) : (
         <TooltipCom
             placement={(props.placement || 'top') as PopperJS.Placement}
-            trigger={'hover'}
+            trigger={props.trigger || 'hover'}
             tooltip={props.text ? props.text : props.content}
+            tooltipShown={props.tooltipShown}
```

Comment:
In case it's not set, does the default behavior change?

Reply:
Nope. Id does not break a thing.